js_modules/dagit/packages/ui/src/components/Dialog.tsx

```diff
@@ -61,6 +61,26 @@ export const DialogBody: React.FC = (props) => {
   );
 };
 
+// The subheadings used in the backfill modals
```

Comment:
Does this need to be in `Dialog`? I'm not sure we have had anything quite like this elsewhere.

This also strikes me as not very dialog-specific -- it's a header and some children, with some spacing -- so I'm not sure this is really the right place for it.

Reply:
Hmm yeah I was kinda torn about this -- it renders these subheadings, and they appear in the backfill modals but not really anywhere else.

I guess for now I'll leave it in the backfill modal and export it into the other backfill modal and we can make it a global shared element if Josh uses this design pattern in more places. 

<img width="304" alt="image" src="https://user-images.githubusercontent.com/1037212/212174282-3d8773d7-034f-4d05-9214-006eb2acd231.png">